### PR TITLE
Added clearer description for feed input

### PR DESCRIPTION
**Pull request by amp-powell**
**Task name**: DotNetCoreCLIV2 and NuGetCommandV2

**Description**: Added a clearer description for feed input to let users know they should just enter the feed name and not a full url

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) Y - Bug 1496559

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/DotNetCoreCLIV2/task.json [hunk 1]
```diff
@@ -191,7 +191,7 @@
         "vstsFeed"
       ],
       "type": "pickList",
-      "label": "Use packages from this Azure Artifacts feed",
+      "label": "Use packages from this Azure Artifacts feed. Please just enter the feed name, not an entire url",
       "defaultValue": "",
       "helpMarkDown": "Include the selected feed in the generated NuGet.config. You must have Azure Artifacts installed and licensed to select a feed here.",
```
Comment (jmyersmsft) [inline Tasks/DotNetCoreCLIV2/task.json:196]: This isn't part of your change, but while you're here:
```suggestion
      "helpMarkDown": "Include the selected feed in the generated NuGet.config.\n\nFormat:\n* For organization-scoped feeds: `FeedName`\n* For project-scoped feeds: `ProjectName/FeedName`",
```
* The deleted part is no longer correct, and hasn't been in ages -- artifacts is included (up to 2GB) with Azure DevOps
* We can be a bit more verbose here than in the label

File: Tasks/DotNetCoreCLIV2/task.json [hunk 2]
```diff
@@ -191,7 +191,7 @@
         "vstsFeed"
       ],
       "type": "pickList",
-      "label": "Use packages from this Azure Artifacts feed",
+      "label": "Use packages from this Azure Artifacts feed. Please just enter the feed name, not an entire url",
```
Comment (jmyersmsft) [inline Tasks/DotNetCoreCLIV2/task.json:194]: Our style generally doesn't use "please" in this context. Just tell the user what they need to do.
```suggestion
      "label": "Use packages from this Azure Artifacts feed. Select from the dropdown or enter [project name/]feed name.",
```

**Review by jmyersmsft — CHANGES_REQUESTED** [review]

File: Tasks/DotNetCoreCLIV2/task.json (hunk 2 above)
Comment (amp-powell) [reply Tasks/DotNetCoreCLIV2/task.json]: Changed this!

File: Tasks/DotNetCoreCLIV2/task.json (hunk 1 above)
Comment (amp-powell) [reply Tasks/DotNetCoreCLIV2/task.json]: Changed this!

**Pull request merged**